build/en/docs/index.njk

```diff
@@ -225,31 +295,28 @@ rootClass: docs
+      <b>Note</b>: Site Kit and Tag Manager do not yet work together. <a href="https://github.com/google/site-kit-wp/issues/2070">Follow progress here</a>.
     </p>
-    <h1 id="Accessibility">Accessibility</h1>
-    <h2>Keyboard shortcuts & navigation</h2>
     <p>
-      View editor keyboard shortcuts by clicking the keyboard icon in the Workspace panel. Or press <code>ctrl</code> + <code>/</code> on Windows and <code>command ⌘</code> + <code>/</code> on Mac.
-    </p>
-      <p class="img-wrap"><amp-img layout="intrinsic" width="212" height="351" src="./img/docs/keyboard.png"></amp-img></p>
-    <h2 id="Using-assistive-text-for-images-video">Using assistive text for images & video</h2>
-    <p>
-      Make your images and videos accessible to users of screen readers by adding assistive text.
-    </p>
-      <p class="img-wrap"><amp-img layout="intrinsic" width="268" height="148" src="./img/docs/25.png"></amp-img></p>
-    <p>
-      For images, insert a description of the image in the "Accessibility" section of the Design panel.
-    </p>
-      <p class="img-wrap"><amp-img layout="intrinsic" width="270" height="375" src="./img/docs/26.png"></amp-img></p>
-    <p>
-      For videos, fill out both the "Title" and "Assistive text" text fields in the "Accessibility section of the Design panel. The title should be a description of the video itself, whereas the assistive text should describe the contents of the video.
+      Web Stories for WordPress includes the following default analytics events:
     </p>
+    <ul class="real-ul">
+      <li><strong>Story Progress:</strong> Fired when a story page becomes visible.</li>
+      <li><strong>Story End:</strong> Fired when the last story page displays to the user. This can be used to measure completion rate.</li>
+      <li><strong>Track Focus State:</strong> Fired when clicking an element that opens a tooltip such as an `<a>` link or twitter embed.</li> 
+      <li><strong>Story Open:</strong> Fired when opening a drawer or dialog inside a story, such as a page attachment.</li>
+      <li><strong>Story Close:</strong> Fired when closing a drawer or dialog inside a story, such as a page attachment.</li>
+      <li><strong>Audio Muted:</strong> Fired when the user initiates an interaction to mute the audio for the current story.</li>
+      <li><strong>Audio Unmuted:</strong> Fired when the user initiates an interaction to unmute the audio for the current story.</li>
+      <li><strong>Page Attachment Enter:</strong> Fired when a user opens a page attachment.</li>
+      <li><strong>Page Attachment Exit:</strong> Fired when a user closes a page attachment.</li>
+      <li><strong>Story Bookend Enter:</strong> Fired when the the story displays the bookend to the user, after the last page of the current story.</li>
+      <li><strong>Story Bookend Exit:</strong> Fired when the user dismisses the bookend.</li>
+    </ul>
```

Comment:
Thanks for listing these!

Should we add the actual event names people can look for in Google Analytics?

For example, the event name for "Track Focus State" is `story_focus`.

Missing from the list:

* Click through - Fired when clicking on a tooltip. (`story_click_through`)

Reply:
Happy to add them in! But I don't see them listed in the sheet?

Reply:
I added it to the sheet now :-)

Reply:
Done! Updated the names to the event actions - which I think makes sense. But I can change it to a table if that makes more sense.